src/gui/properties/trackerlist.h

```diff
@@ -31,60 +31,78 @@
 #ifndef TRACKERLIST_H
 #define TRACKERLIST_H
 
+#include <QClipboard>
+#include <QList>
 #include <QShortcut>
 #include <QTreeWidget>
-#include <QList>
-#include <QClipboard>
 
 #include "propertieswidget.h"
 
-enum TrackerListColumn {COL_TIER, COL_URL, COL_STATUS, COL_RECEIVED, COL_SEEDS, COL_PEERS, COL_DOWNLOADED, COL_MSG};
 #define NB_STICKY_ITEM 3
 
 namespace BitTorrent
 {
     class TorrentHandle;
 }
 
-class TrackerList: public QTreeWidget {
-  Q_OBJECT
-  Q_DISABLE_COPY(TrackerList)
-
-private:
-  PropertiesWidget *properties;
-  QHash<QString, QTreeWidgetItem*> tracker_items;
-  QTreeWidgetItem* dht_item;
-  QTreeWidgetItem* pex_item;
-  QTreeWidgetItem* lsd_item;
-  QShortcut *editHotkey;
-  QShortcut *deleteHotkey;
-  QShortcut *copyHotkey;
+class TrackerList: public QTreeWidget
+{
+    Q_OBJECT
+    Q_DISABLE_COPY(TrackerList)
 
 public:
-  TrackerList(PropertiesWidget *properties);
-  ~TrackerList();
+    enum TrackerListColumn
+    {
+        COL_TIER,
+        COL_URL,
+        COL_STATUS,
+        COL_RECEIVED,
+        COL_SEEDS,
+        COL_PEERS,
+        COL_DOWNLOADED,
+        COL_MSG,
 
```

Comment:
@thalieht:  I thought as much, but I was worried about style consistency.  

If that is not a problem, then you can leave it as is.